web/app/index.html

```diff
@@ -39,6 +40,8 @@
     <script src="bower_components/angular-bootstrap/ui-bootstrap-tpls.js"></script>
     <script src="bower_components/lodash/lodash.js"></script>
     <script src="bower_components/angular-uuid/uuid.min.js"></script>
+    <script src="bower_components/leaflet/dist/leaflet.js"></script>
+    <script src="bower_components/leaflet/dist/leaflet-src.js"></script>
```

Comment:
It seems like only one of these scripts would be needed. Any reason for having both?